### PR TITLE
SOF-1103 Able to create on next take transition action trigger

### PR DESCRIPTION
**Pull request by LindvedKrvang**
All transition now also have a NEXT_TAKE_WITH_TRANSITION actionTrigger created.

Refactored RundownAdLibActions to use less duplication

### CODE REVIEW

**Review by RasmusAlbrektsen — APPROVED** [review]
Lovely refactor :-D

File: src/tv2-common/helpers/rundownAdLibActions.ts [hunk 1]
```diff
@@ -128,36 +103,67 @@ export function ParseTransitionString(transitionString: string): ActionTakeWithT
...
-	jingle: string,
-	alphaAtStart: number | undefined,
-	duration: number | undefined,
-	alphaAtEnd: number | undefined
+	transitionValues: TransitionValues,
+	adlibTag: AdlibTags
 ): IBlueprintActionManifest {
 	const tag = GetTagForTransition(userData.variant)
-	const isEffekt = !!label.match(/^\d+$/)
+	const isEffekt = !!transitionValues.label.match(/^\d+$/)
```
Comment (ianshade) [inline src/tv2-common/helpers/rundownAdLibActions.ts:139]: Could use `test`
```suggestion
	const isEffekt = /^\d+$/.test(transitionValues.label)
```

File: src/tv2-common/helpers/rundownAdLibActions.ts [hunk 2]
```diff
@@ -128,36 +103,67 @@ export function ParseTransitionString(transitionString: string): ActionTakeWithT
...
-			tags: [AdlibTags.ADLIB_STATIC_BUTTON, AdlibTags.ADLIB_TAKE_WITH_TRANSITION],
+			tags: [AdlibTags.ADLIB_STATIC_BUTTON, adlibTag],
 			currentPieceTags: [tag],
 			nextPieceTags: [tag],
 			content:
-				isEffekt || !!/^MIX ?\d+$/i.test(label) || !!/^CUT$/i.test(label) || !!/^DIP ?\d+$/i.test(label)
+				isEffekt ||
+				!!/^MIX ?\d+$/i.test(transitionValues.label) ||
+				!!/^CUT$/i.test(transitionValues.label) ||
+				!!/^DIP ?\d+$/i.test(transitionValues.label)
```
Comment (ianshade) [inline src/tv2-common/helpers/rundownAdLibActions.ts:158]: `test` already returns a boolean.
```suggestion
				/^MIX ?\d+$/i.test(transitionValues.label) ||
				/^CUT$/i.test(transitionValues.label) ||
				/^DIP ?\d+$/i.test(transitionValues.label)
```

**Review by ianshade — APPROVED** [review]
Looks good!

File: src/tv2-common/helpers/rundownAdLibActions.ts (hunk 1 above)
Comment (LindvedKrvang) [reply src/tv2-common/helpers/rundownAdLibActions.ts]: Leftover from before refactor - has been updated

File: src/tv2-common/helpers/rundownAdLibActions.ts (hunk 2 above)
Comment (LindvedKrvang) [reply src/tv2-common/helpers/rundownAdLibActions.ts]: Leftover from before refactor - has been updated

**Pull request merged**